mypy/util.py

```diff
@@ -906,3 +919,20 @@ def quote_docstring(docstr: str) -> str:
         return f"''{docstr_repr}''"
     else:
         return f'""{docstr_repr}""'
+
+
+_default = object()
+
+
+@overload
+def getattr(value: object, name: str) -> Any: ...
+@overload
+def getattr(value: object, name: str, default: T) -> Any | T: ...
+def getattr(value, name, default: object = _default):
+    if default is _default:
+        return builtins_getattr(value, name)
+    return builtins_getattr(value, name, default)
```

Comment:
Isn't this exactly the same as `builtins.getattr`...? 

I agree that the [typeshed annotations for it](https://github.com/python/typeshed/blob/main/stdlib/builtins.pyi#L1438-L1453) are retarded, but that's already fixed in your own `builtins.pyi`, right?

Reply:
yeah it's the same, just to become `Any` for stuff that's upstream code